lsp/nls/src/usage.rs

```diff
@@ -37,7 +37,8 @@ pub struct UsageLookup {
     // This could be made more general (we might want to map arbitrary positions to
     // environments) but its enough for us now.
     def_table: HashMap<RawSpan, Environment>,
-    usage_table: HashMap<LocIdent, Vec<LocIdent>>,
+    // Maps from spans of idents to the places where they are referenced.
+    usage_table: HashMap<RawSpan, Vec<LocIdent>>,
```

Comment:
Just to make sure to understand: before, the table was hashing on _both the identifier and the position_ ? And now we just use the position, which should be entirely the same for located idents (that is, hashing on the ident as well was useless anyway) ?

Reply:
Exactly. In fact, we are still hashing a little too much, because the `RawSpan` contains an unnecessary `FileId`...